### PR TITLE
feat: noPadding on grid

### LINKED ISSUE #23: Make possible to skip gutter around columns
Sometimes it's desired to create a grid without the outer gutters. This means there's no gutter by the left side of the first column and no gutter by the right side of the last column. Right now we've had to create our own lightweight grid to conform to the design requirements.

### PR DESCRIPTION
**Pull request by mlenser**
Closes #23

### CODE REVIEW

File: src/grid/index.js [hunk 1]
```diff
@@ -12,11 +12,13 @@ function Grid({
   style,
   children,
   componentClass: ComponentClass,
+  noPadding,
 }) {
   const className = cn(
     {
       [classes.grid]: !fluid,
       [classes.gridFluid]: fluid,
+      [classes['noPadding']]: noPadding,
```
Comment (iamstarkov) [inline src/grid/index.js]: dot notation doesnt fit?

**Comment by iamstarkov** [comment]
looks good, but can we use dot notation to conform rest of the codebase?

File: src/grid/index.js (hunk 1 above)
Comment (mlenser) [reply src/grid/index.js]: I just copied how it is on the column. I can change both to use dot notation.

**Comment by mlenser** [comment]
@iamstarkov both col and grid use dot notation now.

**Pull request merged**